src/main/java/com/kaziamyr/onlinebookstore/repository/impl/BookRepositoryImpl.java

```diff
@@ -1,5 +1,6 @@
 package com.kaziamyr.onlinebookstore.repository.impl;
 
+import com.kaziamyr.onlinebookstore.exception.EntityNotFoundException;
```

Comment:
You don't need the Autowired annotation, spring understands it automatically. Anyway it is better to use the annotation that we have studied - @RequiredArgsConstructor